src/api/directives.ts

```diff
@@ -74,4 +79,27 @@ const submitDirective = async (
     }
 };
 
-export { exchangeBearerToken, submitDirective };
+const getApplieDirectives = (type: keyof typeof DIRECTIVES, userId: string) => {
```

Comment:
>`(type: keyof keyof DIRECTIVES`

Since `type` is a keyword, I would suggesting renaming this input argument. Also, I have never seen `keyof` used in succession before. Interesting.

Reply:
I considered that - but they are stored as `type` so sticking with it.